utilities/DumpUnitary/Operations.qs

```diff
@@ -38,4 +38,17 @@ namespace Quantum.DumpUnitary
 
         DumpUnitaryToFiles(N, unitary);
     }
+
+    operation TestUnitary (qs : Qubit[]) : Unit {
```

Comment:
I would add some visual distinction between the code that is the main part of the tool (and that is supposed to be modified by the user) and the test code (they look pretty similar). Maybe add a line of --- and a comment under it saying "The code below this line is part of the unit test; you do not need to modify it to run the tool on your unitary"?